pkg/util/utils.go

```diff
@@ -723,3 +726,102 @@ func SocketPath() (string, error) {
 	// Glue the socket path together
 	return filepath.Join(xdg, "podman", "podman.sock"), nil
 }
+
+// ImageScpOptions provide options for securely copying images to and from a remote host
```

Comment:
Why has this been moved to `pkg/utils`?  Since there's only one caller I think it could continue living in `cmd/podman/...`

Reply:
@vrothberg this is here because entities and utils cannot be imported into one another. It it a temp struct used to convert back to the entities `ImageSCPOptions` I have tried a bunch of other configs and they all break the import cycle

Reply:
To me this sounds like too many indirections.  Let's try to avoid having too many abstractions. Can't we just keep the code in `pkg/domain/infra/abi/images.go`?

Reply:
and `ParseImageSCPArg` has to be in utils for testing purposes. Either way I am going to have an extra struct sitting somewhere

Reply:
w/ this redesign we wanted to have unit tests for the image scp args since the integration tests dont really work with this feature. I originally had this in pkg/domain/infra/abi/images.go @vrothberg

Reply:
Why can´t the unit tests live somewhere else? I would naively assume the parsing code could even live in `cmd/podman/images`

Reply:
I thought unit tests were only for structs and functions within utils

Reply:
Ah that explains why you moved it to `utils`. We can add unit tests to all packages. The implicit magic/convention in Go is to add `_test.go` file and that's it.

Only some packages are skipped during `make localunit`, see https://github.com/containers/podman/blob/main/Makefile#L524.

Reply:
oh cool! I will change that around this morning and re-push. Thanks for the help